backend/src/packages/subscriptions/libs/enums/table-columns/subscriptions-table-column.enum.ts

```diff
@@ -0,0 +1,8 @@
+const SubscriptionsTableColumn = {
```

Comment:
why do we have two SubscriptionsTableColumn enums?

Reply:
I've noticed that we have this tendency with other tables as well, that's why I decided to do so. I will remove the one which is in the database package

Reply:
thank you for noticing, I will remove unnecessary enums